src/Microsoft.AspNet.Mvc.Razor/RazorViewEngine.cs

```diff
@@ -228,6 +228,29 @@ public virtual IEnumerable<string> AreaViewLocationFormats
             return new RazorPageResult(pageName, searchedLocations);
         }
 
+        private string GetNormalizedRouteValue(string key, ActionContext context)
```

Comment:
This is absolutely worth documenting in the comments what the bug we're fixing is here and why it's so important.